### PR TITLE
Improve docs CSS

### PR DESCRIPTION
**Pull request by celestehorgan**
A few small changes to the CSS to make things look better!

### CODE REVIEW

File: assets/sass/docs.sass [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $docs-sidebar-logo-width: 30%
-$docs-sidebar-margin: 1.75rem
+$docs-sidebar-margin: .75rem
```
Comment (lucperkins) [inline assets/sass/docs.sass]: This feels narrow to me, especially on top. Could we do something more like `1.25rem 1.5rem`?

File: assets/sass/docs.sass [hunk 2]
```diff
@@ -100,9 +101,26 @@ html, body
...
+        h2
+          font-size: 1.602rem
+        h3
+          font-size: 1.424rem
+        h4
+          font-size: 1.266rem
+        h5
+          font-size: 1.125rem
+        h2, h3, h4, h5, h6
+          color: $vitess-orange
```
Comment (lucperkins) [inline assets/sass/docs.sass:115]: This orange feels a bit light for headers. Could you darken that 10% and see how it comes out?

**Review by raunaqgupta — COMMENTED** [review]
1. I find the reduced padding in the sidebar constricting as it now feels the list items are scrunched upto the left and not properly utilizing all of the width as before. One way I can see this working is if the Vitess logo was also left justified instead of centered as it is currently. Although it's size would have to be adjusted because it feels too big now with respect to the padding.

2. The removal of padding on the landing page's content along with the border right above it looks good.

3. Unsure about the `is-radiusless` addition? Can you provide a before/after screenshot?

**Comment by celestehorgan** [comment]
@raunaqgupta 

Thanks for your comments!

> I find the reduced padding in the sidebar constricting as it now feels the list items are scrunched upto the left and not properly utilizing all of the width as before. 

I adjusted the padding because on smaller resolutions it feels too big/too spaced out. I'm not attached to the decision, however, and am happy to reverse it.

> The removal of padding on the landing page's content along with the border right above it looks good.

Thanks :)

> Unsure about the `is-radiusless` addition? Can you provide a before/after screenshot?
This affects admonition callouts, and is a [helper class](https://bulma.io/documentation/modifiers/helpers/) from Bulma, the CSS framework this site uses. As the name suggests, it removes the border radius from admonitions, bringing them more in-line with the sharp cornered overall look. [Before](https://vitess.io/docs/contributing/build-on-centos/) and [After](https://deploy-preview-410--admiring-colden-94a5f1.netlify.com/docs/contributing/build-on-centos/).

**Review by lucperkins — APPROVED** [review]

**Pull request merged**